web/src/styles/base/border-colors.scss

```diff
@@ -30,3 +30,7 @@
 .border-right-base {
   border-right-color: $base !important;
 }
+
+.border-base-lightest {
+  border-right-color: $base-lightest !important;
```

Comment:
`border-base-lightest` is already a class that exists from NJWDS, right? Should we be overriding it to just be border-right? Why not make a new class name specific to setting the right-border?

Reply:
We don't need this styling, deleting.